bedrock/products/templates/products/vpn/download.html

```diff
@@ -37,121 +39,219 @@
+            {{ ftl('vpn-download-get-mozilla-vpn') }}
+            </a>
+          </div>
+        </div>
+        <!-- iOS -->
+        <div class="primary-platform ios">
+          <div class="platform-image">
+            <img src="{{ static('img/products/vpn/download/platform-ios.svg') }}" alt="">
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-download-for-ios') }}</h2>
+            <p class="current-platform-lede">{{ ftl('vpn-download-based-on-your') }}</p>
+            <p>{{ ftl('vpn-download-version-requirements', version='13.0') }}</p>
+            <a href="{{ ios_download_url }}" data-cta-type="button" data-cta-text="VPN Download (iOS)" class="platform-download-link">
+              <img src="{{ static('img/products/vpn/download/apple-app-store-badge.svg') }}" alt=" {{ ftl('vpn-download-get-mozilla-vpn') }}">
+            </a>
+          </div>
+        </div>
+        <!-- Android -->
+        <div class="primary-platform android">
+          <div class="platform-image">
+            <img src="{{ static('img/products/vpn/download/platform-android.svg') }}" alt="">
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-download-for-android') }}</h2>
+            <p>{{ ftl('vpn-download-version-requirements', version='8.0') }}</p>
+            <p>{{ ftl('vpn-download-version-requirements', version='13.0') }}</p>
+            <a class="platform-download-link" href="{{ android_download_url }}" data-cta-type="button" data-cta-text="VPN Download (Android)" target="_blank" rel="noopener noreferrer">
+              <img src="{{ static('img/products/vpn/download/google-play-badge.png') }}" alt=" {{ ftl('vpn-download-get-mozilla-vpn') }}">
+            </a>
+          </div>
+        </div>
+      </div>
+      <ul class="vpn-download-secondary-list">
+        <!-- Windows -->
+        <li class="secondary-platform windows">
+          <div class="platform-image">
+            <img src="{{ static('img/products/vpn/download/platform-windows.svg') }}" alt="">
+          </div>
+          <div class="platform-body">
+            <h2>{{ ftl('vpn-download-windows') }}</h2>
+            <p>{{ ftl('vpn-download-for-windows-requirements') }}</p>
+          </div>
+          <a href="{{ windows_download_url }}" data-cta-type="button" data-cta-text="VPN Download (Windows)" class="platform-download-link">
+            <svg class="platform-download-icon" width="42" height="42" fill="none" xmlns="http://www.w3.org/2000/svg">
+              <circle cx="21" cy="21" r="20" stroke="#000" stroke-width="2"/>
+              <path d="M20.293 25.707a1 1 0 0 0 1.414 0l5-5a1 1 0 0 0-1.414-1.414L22 22.586V14a1 1 0 0 0-2 0v8.586l-3.293-3.293a1 1 0 0 0-1.414 1.414l5 5ZM26 27H16a1 1 0 0 0 0 2h10a1 1 0 0 0 0-2Z" fill="#0C0C0D"/>
+            </svg>
           </a>
```

Comment:
Great idea. I was able to achieve the hover state by just adding a `background-color` and making the `border-radius: 50%` to match the silhouette of the svg.  It resulted in cleaner code and was easy to read (avoids the developer from seeing the inline svg on the html file).
I also tested making a new svg file of this icon to match its hover state (grey background), but since I was able to achieve the wanted result quickly within CSS, I didn't feel the need to have the hover state show the grey counterpart of this icon.

What do you think?